components/psa/services/psa_prot_internal_storage/COMPONENT_PSA_SRV_EMUL/psa_prot_internal_storage.cpp

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#include "psa_prot_internal_storage.h"
+#include "pits_impl.h"
+#include "kv_config.h"
+#include "mbed_error.h"
+
+#define PSA_ITS_EMUL_PID        0xFFFFFFFF  // In EMUL world, there is no real partitioning, which makes the source partition irrelevant.
```

Comment:
Smaller PIDs will result in shorter file names -> less flash required to store them.
PID in PSA emulation is not really important.
Please use `0` as PID

Reply:
Since 0 is illegal pid in psa terms, I prefer using 1.
Although it doesn't matter now, this will make it proof in case of future validations on pid.

Reply:
all number you pick less then 0x3F will have same footprint.
from PSA point of view it is still bad as NSPE PIDs should be in negative range...